packages/vuetify/src/components/VStepper/VStepper.ts

```diff
@@ -30,16 +41,16 @@ export default {
 
   data () {
     return {
-      inputValue: null,
+      inputValue: null as any,
```

Comment:
Maybe `string | number | null` ?

Reply:
not a big deal, this is getting a full rewrite in 2.0